Resources/config/factories.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+
+        <service id="cmf_resource.factory.filesystem" class="Symfony\Cmf\Bundle\ResourceBundle\Factory\FilesystemFactory">
+            <tag name="cmf_resource.repository_factory" alias="filesystem" />
+        </service>
+
+        <service id="cmf_resource.factory.doctrine_phpcr" class="Symfony\Cmf\Bundle\ResourceBundle\Factory\PhpcrFactory">
```

Comment:
We do not have to devide services definitions by its persistence layer in this bundle yet, right?